lldb/include/lldb/Core/Debugger.h

```diff
@@ -593,6 +593,7 @@ class Debugger : public std::enable_shared_from_this<Debugger>,
   friend class CommandInterpreter;
   friend class REPL;
   friend class Progress;
+  friend class ProgressManager;
```

Comment:
I don't think you need this (anymore)?

Reply:
`Debugger::ReportProgress` is protected, so to my understanding if we call that function from `ProgressManager::ReportProgress` without having it as a friend it leads to a compile error.